StratisCore.UI/src/app/setup/create/create.component.ts

```diff
@@ -22,11 +22,13 @@ export class CreateComponent implements OnInit {
   }
 
   public createWalletForm: FormGroup;
+  public sidechainEnabled: Boolean;
```

Comment:
I think the best practive is not to use Boolean (the one with capital B), because
```
var falseObj = new Boolean(false);
if (falseObj) {
    console.log('our intent may not be to get here');
}
```
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html